R/grob-dotstack.r

```diff
@@ -13,14 +13,16 @@ dotstackGrob <- function(
         y <- unit(y, default.units)
     if (!is.unit(dotdia))
         dotdia <- unit(dotdia, default.units)
-    if (attr(dotdia,"unit") != "npc")
+    if (!is_npc(dotdia)) # Only cross-version reliable way to check the unit of a unit object
```

Comment:
The comment should probably be moved to where the function `is_npc()` is defined.

Reply:
Otherwise looks all good.